### PR TITLE
[RFR] added basic create project functionality with a sample test

### PR DESCRIPTION
**Pull request by stillalearner**

### CODE REVIEW

**Comment by abrugaro** [comment]
LGTM!
There are some minor lint corrections but they can be solved later by adding [prettier](https://prettier.io/) or something like that to the project

File: cypress/e2e/models/projects/projects.ts [hunk 1]
```diff
@@ -1,3 +1,173 @@
...
+    selectCard(target: string): void {
+
+        if(target.includes("eap")){
+
+            //cy.get(targetOptions).contains("eap7").click({force: true});
+            clickByText(targetOptions, "eap");
+            clickByText("button", target);
+
+
+        } else if(target.includes("openjdk")){
```
Comment (sshveta) [inline cypress/e2e/models/projects/projects.ts:99]: We do not need so many if else . 
We can just do 



clickByText(targetOptions, target);
clickByText("button", target);

File: cypress/e2e/tests/test_project.cy.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { login } from "../../utils/utils";
+import { Projects } from "../models/projects/projects";
```
Comment (sshveta) [inline cypress/e2e/tests/test_project.cy.ts:2]: File name can be project.test.ts instead of test_project.cy.ts

**Review by nachandr — COMMENTED** [review]
Sometimes, the indentation of statements inside else may affect which lines are included.
Just a suggestion - you could use "[prettier](https://www.npmjs.com/package/prettier)" library to format the code and avoid such indentation issues. You could include 'Prettier' as a dependency in the package.json file.

File: cypress/utils/utils.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    cy.wait(SEC);
+}
+
+export function click(fieldId: string, isForced = true): void {
+    cy.get(fieldId, { timeout: 120 * SEC }).click({ force: isForced });
+}
+
+//TODO: Login functionality for secure URL
+export function login(username?, password?: string): void {
+    cy.visit(tackleUiUrl, { timeout: 120 * SEC });
```
Comment (sshveta) [inline cypress/utils/utils.ts:33]: It will be windupURL

**Review by sshveta — CHANGES_REQUESTED** [review]

File: cypress/e2e/models/projects/projects.ts [hunk 4]
```diff
@@ -1,3 +1,173 @@
...
+    addApplications(apps: string[]): void {
+
+        apps.forEach(app => importApp(app));
+
+        clickByText(primaryButton, next);
+
+    }
+
+    selectTarget(targets: string[]): void{
+        clickByText("h4", eapCard);
```
[the remark this answers is not in the document]
Comment (stillalearner) [reply cypress/e2e/models/projects/projects.ts:83]: when we move to "select target" section, by default eap card is always selected, to remove that first, I am clicking on it.

File: cypress/e2e/models/projects/projects.ts (hunk 1 above)
Comment (stillalearner) [reply cypress/e2e/models/projects/projects.ts:99]: In the first clickByText, target cannot be used, as target can be "eap8", which is not visible on the page, until the dropdown is clicked.

Similarly for azure, if target is azure-aks, its not visible until you click on azure-appservice dropdown.

Comment (stillalearner) [reply cypress/e2e/models/projects/projects.ts:99]: so, had to keep it separate, this was the most challenging part in this PR :laughing:

File: cypress/e2e/fixtures/custom.Test1rules.rhamt.xml [hunk 5]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
```
[the remark this answers is not in the document]
Comment (stillalearner) [reply cypress/e2e/fixtures/custom.Test1rules.rhamt.xml:1]: done

File: cypress/e2e/tests/test_project.cy.ts (hunk 2 above)
Comment (stillalearner) [reply cypress/e2e/tests/test_project.cy.ts:2]: done

File: cypress/utils/utils.ts (hunk 3 above)
Comment (stillalearner) [reply cypress/utils/utils.ts:33]: done

**Review by sshveta — APPROVED** [review]

**Pull request merged**